fitbenchmarking/results_output.py

```diff
@@ -99,6 +99,7 @@ def print_group_results_tables(minimizers, results_per_test, problems_obj, group
                                metric_type=FILENAME_SUFFIX_ACCURACY, file_extension=FILENAME_EXT_HTML)
 
         # print out accuracy summary table for this group of fit problems
+        # THIS IS BROKEN
```

Comment:
what is broken?

Reply:
The accuracy summary table.

Reply:
The accuracy summary table.

Reply:
what is it doing and what should it be doing?

Reply:
It should output a table with the most accurate minimser etc., instead it outputs gibberish.